modules/rules_java/metadata.json

```diff
@@ -1,45 +1,41 @@
 {
-  "homepage": "https://github.com/bazelbuild/rules_java",
-  "maintainers": [
-    {
-      "email": "bcr-maintainers@bazel.build",
-      "name": "No Maintainer Specified"
-    }
-  ],
-  "versions": [
-    "4.0.0",
-    "5.0.0",
-    "5.1.0",
-    "5.3.5",
-    "5.4.0",
-    "5.4.1",
-    "5.5.0",
-    "5.5.1",
-    "6.0.0",
-    "6.1.0",
-    "6.1.1",
-    "6.2.2",
-    "6.3.0",
-    "6.3.1",
-    "6.3.2",
-    "6.3.3",
-    "6.4.0",
-    "6.5.0",
-    "6.5.1",
-    "6.5.2",
-    "7.0.0",
-    "7.0.6",
-    "7.1.0",
-    "7.2.0",
-    "7.3.0",
-    "7.3.1",
-    "7.3.2"
-  ],
-  "yanked_versions": {
-    "6.3.3": "6.3.3 is a broken version.",
-    "7.0.0": "7.0.0 is a broken version."
-  },
-  "repository": [
-    "github:bazelbuild/rules_java"
-  ]
+    "homepage": "https://github.com/bazelbuild/rules_java",
+    "maintainers": [],
```

Comment:
@hvadehra Could you add yourself here?

Reply:
@keertk You could also add yourself to this list, better to have two maintainers listed.